_IBC/archway-osmosis.json

```diff
@@ -55,6 +55,21 @@
       "tags": {
         "status": "live"
       }
+    },
+    {
+      "chain_1": {
+        "channel_id": "channel-*",
+        "port_id": "wasm.*"
+      },
+      "chain_2": {
+        "channel_id": "channel-*",
+        "port_id": "icahost"
+      },
+      "ordering": "ordered",
+      "version": "*",
```

Comment:
why not use ics27-1?

Reply:
For ICA, the version string contains more metadata and `ics27-1` is only a part inside the full string. An example of ICA version is like this.

```
"{\"version\":\"ics27-1\",\"controller_connection_id\":\"connection-123\",\"host_connection_id\":\"connection-1172\",\"address\":\"osmo1a9n50063atnvjmh70fzkc5pvh865tm4q4gg6k48gzghp0dvjufksw3g3zc\",\"encoding\":\"proto3\",\"tx_type\":\"sdk_multi_msg\"}", "port_id": "wasm.neutron1ftql28jnc79936w5qx0kyra4c8k3tqtupq8gxvy5aq2udxpzddcsdq3xr0"
```

Reply:
network repo complies with upstream chain registry [standards](https://github.com/cosmos/chain-registry/blob/0bae8f2641f28db67eef674d7bd6cf0319933cae/ibc_data.schema.json#L48), now they dont seem to have well defined requirements for version definition but conventions seems to be just "ics27-1"

Reply:
Just `ics27-1` is not enough. It should include all these metadata. Incase if you didn't notice, the example value I gave in previous comment is a string which is the **version** for ICA.

https://github.com/cosmos/ibc/blob/main/spec/app/ics-027-interchain-accounts/README.md

> The caller must provide the complete channel version. This MUST include the ICA version with complete metadata and it MAY include versions of other middleware that is wrapping ICA on both sides of the channel.

https://github.com/cosmos/ibc/blob/main/spec/app/ics-027-interchain-accounts/README.md#metadata-negotiation
> In the case of interchain accounts, the channel version will be a string of a JSON struct containing all the relevant metadata intended to be relayed to the counterparty during the channel handshake step (see summary below).

Reply:
yes the IBC spec expects the implementation to provide complete version, but its not necessary for publishing the channel here in the repo. On this repo we we only need to worry about [json schema](https://github.com/archway-network/networks/blob/main/ibc_data.schema.json) to publish channels, which follows upstream [chain registry schema](https://github.com/cosmos/chain-registry/blob/master/ibc_data.schema.json). These json schemas are more lenient than the actual IBC spec. This helps us easily keep this networks repo in sync with chain registry.  And its seems chain-registry accepts simple `ics27-1` e.g. [here](https://github.com/cosmos/chain-registry/blob/0bae8f2641f28db67eef674d7bd6cf0319933cae/_IBC/juno-stride.json#L74)

Reply:
I thought this acts like a filter for packets and if the value here doesn't match the actual value, it won't relay. Anyway I updated it now. Thanks!

Reply:
take care with ICS27, an issue exist with IBC when a channel uses ICS27 and PFM hooks are enable

Reply:
@Galadrin What is the issue can you share more please?

Reply:
It would seem that the current PFM doesn’t handle `IncentivizedAcknowledgment`, which can result in the PFM module failing due to unmarshal failure of acknowledgment.

we discovered this issue on Injective and reproduced on Migaloo.